modules/gitops-configuring-namespace-scoped-argo-rollouts-installation.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// * argo_rollouts/enable-support-for-namespace-scoped-argo-rollouts-installation.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="configuring-namespace-scoped-argo-rollouts-installation_{context}"]
+= Configuring a namespace-scoped Argo Rollouts installation
+
+To install a namespace-scoped instance of Argo Rollouts, complete the following steps:
```

Comment:
Shouldn't end in a colon, and module title should match the intro/shortdesc. See https://redhat-documentation.github.io/modular-docs/#con-creating-procedure-modules_writing-mod-docs.

Reply:
Also, am I configuring an installation or installing something?

Reply:
Fixed.